Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", ] as String[]
+def paths = []
+
+// using mesos node because it's a lightweight alpine docker image instead of full VM
+node('mesos') {
+  stage("Verify author") {
+    user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#tools-notify')
+  }
+}
+
+node('mesos-ubuntu') {
+  def shcmd = { String command ->
+      output = sh(script: command, returnStdout: true).trim()
+      println(output)
+      return output
+  }
+
+  checkout scm
+  stage("Get changeset") {
+    shcmd("git config remote.origin.fetch '+refs/heads/*:refs/remotes/origin/*'")
+    shcmd('git fetch --all')
+    branch = shcmd('git ls-remote --heads origin | grep $(git rev-parse HEAD) | cut -d / -f 3')
+    shcmd('git checkout master')
```

Comment:
This can be removed, I think. Not sure what this checkout is providing as the next line is also a checkout.

Reply:
@colin-msphere  creates the local branch master and sets its upstream to origin/master, which is necessary for the next step "git rebase master". "git rebase origin/master" had given me issues.

Reply:
@colin-msphere Actually I just remembered this was only a problem for git merge-base and not rebase! I will switch to origin/master